google/cloud/pubsub/CMakeLists.txt

```diff
@@ -379,11 +379,17 @@ install(
             NAMELINK_ONLY
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
             COMPONENT google_cloud_cpp_development)
-
 google_cloud_cpp_install_headers("google_cloud_cpp_pubsub"
                                  "include/google/cloud/pubsub")
-google_cloud_cpp_install_headers("google_cloud_cpp_pubsub_mocks"
-                                 "include/google/cloud/pubsub")
+
+install(
+    TARGETS google_cloud_cpp_pubsub_mocks
+    EXPORT pubsub-targets
+    COMPONENT google_cloud_cpp_development)
+install(
+    FILES ${google_cloud_cpp_pubsub_mocks_hdrs}
+    DESTINATION "include/google/cloud/pubsub/mocks"
+    COMPONENT google_cloud_cpp_development)
 
 google_cloud_cpp_add_pkgconfig(
     pubsub
```

Comment:
Q: should we have one of these for `pubsub_mocks` ?

Reply:
Yup.